src/hooks/useUsersAttempts.js

```diff
@@ -72,13 +67,10 @@ const useUsersAttempts = ({ wordLength, correctWord, letters, setLoading }) => {
         setWordDate(today);
         setUsersAttempts([Array(wordLength).fill('')]);
         setRoundsResults([Array(wordLength).fill('')]);
-        try {
-          const q = query(
-            collection(firebaseDb, DAILY_RESULTS),
-            where('user.email', '==', currentUser),
-            where('date', '==', today)
-          );
-          const docs = await getDocs(q);
+        const { docs, error } = await getDailyResults(currentUser, today);
```

Comment:
please use  try/catch and include finally, in the finally section you could setLoading(false)  and remove it when there isn't an error

Reply:
I actually have the try/catch inside the getDailyResults